### DISCUSSION TITLE
!URGEND! Help for footer Duplication

### DISCUSSION THREAD

**Discussion by ghost** [GrapesJS/grapesjs#6006] [Q&A]
Hey Guys, i want to use my footer for every page that i have created. Is the webpage Website Builder (https://studio.grapesjs.com/) capable of doing that? Here i quick Video what i try to archieve. Copying the footer and dropping it on my other page, opened in a different tab. But that doesnt work the way i try to do it. P.S i am not capable of using the repository, i tried multiple thound times but i cant do it.

Thanks for your Help

https://github.com/user-attachments/assets/7bce4a89-af00-4e6b-86f4-3727837af4a3

**Comment by mosh-tudor** [comment]
@marlonivo There are solutions for copying components between pages or/and creating global components (symbols), but they are not available out of the box. You need to "google it".

**Reply by ghost** [reply]
What do you think, I'll have to google it. If you know it's possible, just send me the source. If I had found it via a search engine, I wouldn't bother writing this question here....
